src/Dapr.Actors/Runtime/ActorMethodContext.cs

```diff
@@ -14,10 +18,20 @@ public struct ActorMethodContext
         private readonly string actorMethodName;
         private readonly ActorCallType actorCallType;
 
+        private readonly MethodInfo callback;
+
         private ActorMethodContext(string methodName, ActorCallType callType)
         {
             this.actorMethodName = methodName;
             this.actorCallType = callType;
+            this.callback = null;
+        }
+
+        private ActorMethodContext(Type callback, string methodName)
+        {
+            this.actorMethodName = "";
+            this.actorCallType = ActorCallType.ActorInterfaceMethod;
+            this.callback = callback.GetMethod(methodName);
```

Comment:
Is this really the right logic (`GetMethod)`? Most cases where reflection is happening on user code are a lot more specialized. For instance if this represents Actor interface method, this is definitely not the right logic.

It feels like this code needs a refactor to handle new capabilities - or maybe it shouldn't be shared between timers and actor methods.

Reply:
removed....was unused